src/main/java/com/duckblade/osrs/toa/features/tomb/RewardSarcophagus.java

```diff
@@ -0,0 +1,277 @@
+/**
+ * BSD 2-Clause License
+ * <p>
+ * Copyright (c) 2023, rdutta
+ * Copyright (c) 2020, AnkouOSRS
```

Comment:
Can you provide me with a link of where this attribution comes from so I can confirm the license?

Reply:
https://github.com/AnkouOSRS/cox-light-colors/blob/master/LICENSE
https://github.com/AnkouOSRS/cox-light-colors/blob/master/src/main/java/com/coxlightcolors/CoxLightColorsPlugin.java